stdlib/@tests/stubtest_allowlists/common.txt

```diff
@@ -60,6 +60,17 @@ io.FileIO.seek
 io.StringIO.seek
 io.StringIO.truncate
 io.TextIOWrapper.truncate
+_io.BufferedRandom.truncate
+_io.BufferedReader.seek
+_io.BufferedReader.truncate
+_io.BufferedWriter.seek
+_io.BufferedWriter.truncate
+_io.BytesIO.readlines
+_io.BytesIO.seek  # Parameter name for a positional-only param differs from its name in the inherited method
```

Comment:
This sounds like something we can fix; mypy shouldn't complain about name mismatches in pos-only params.

Reply:
Agreed, but just to note that the new allow-list entries are just copies of the existing `io.*` allow-list entries, updated for `_io.*`